docs/reference/remap/is_blank.cue

```diff
@@ -0,0 +1,36 @@
+package metadata
+
+remap: functions: is_blank: {
+	arguments: [
+		{
+			name:        "value"
+			description: "The value to check for blankness"
+			required:    true
+			type: ["string", "null"]
+		},
+	]
+	return: ["boolean"]
+	category: "Parse"
+	description: #"""
+		Determines whether the provided value should be considered blank, where blank is defined as
+		one of the following:
+
+		* `""`
+		* `" "`
```

Comment:
I see we only allow one space, should we just allow any number of any whitespace characters? I think that would be more consistent.

Reply:
Okay, I've updated to match on any whitespace-only string